src/module/actor/character/document.ts

```diff
@@ -730,7 +730,7 @@ class CharacterPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e
             entry.system.ability.value = baseStat.ability ?? entry.system.ability.value;
             entry.system.proficiency.value = Math.max(entry.rank, baseStat.rank ?? 0) as ZeroToFour;
             entry.statistic = baseStat.extend({
-                slug: entry.slug ?? sluggify(entry.name),
+                slug: entry.slug ?? sluggify(`${entry.name}-spellcasting`),
```

Comment:
arcane-spontaneous-spells-spellcasting?

Reply:
or athletics-spellcasting depending on what they name it

Reply:
Outta have a neater way to handle it I imagine, but I can't imagine it being trivial to make it check last for a middle-of-the-road one.